signac/project.py

```diff
@@ -20,17 +20,17 @@
 from tempfile import TemporaryDirectory
 from threading import RLock
 
+from ._config import (
+    Config,
+    get_project_config_fn,
+    load_config,
+    locate_config_dir,
+    raise_if_older_schema,
+    read_config_file,
```

Comment:
We are now expose these without underscores then to the users though `signac.project`. Is the idea here that most users interact with signac only through the top module?

Reply:
That is a good point, and somewhat related to #876. This issue isn't unique. There are many places where we have modules marked as internal. Should we also require that their contents are always prefixed with an underscore? I don't really mind that, so I can revert https://github.com/glotzerlab/signac/pull/877/commits/f94a2a7f1e77b73f17f3df708b4efad81f2c2c83 if you think so.

Reply:
Reverted.